sql3/planner/oppqlaggregate.go

```diff
@@ -64,7 +64,7 @@ func (p *PlanOpPQLAggregate) Schema() types.Schema {
 	s := &types.PlannerColumn{
 		ColumnName:   "",
 		RelationName: "",
-		Type:         p.aggregate.FirstChildExpr().Type(),
+		Type:         p.aggregate.Type(),
```

Comment:
Why Type() on line 67 but String() on 45?

Reply:
They are two different things - on line 45 we are just populating the plan, we don't need the whole type, just the text description. On 67 we are returning the Schema for the operator - this needs the complex type.